templates/security/livepatch.html

```diff
@@ -309,10 +309,10 @@ <h3 class="p-card__title">Free for personal use</h3>
         <a href="/pro" class="p-button--positive">Get your free subscription</a>
       </div>
       <div class="p-card col-6">
-        <h3 class="p-card__title">Part of Ubuntu Pro and Advantage</h3>
+        <h3 class="p-card__title">Part of Ubuntu Pro</h3>
         <hr>
-        <p>Get Livepatch with Ubuntu Pro or your Ubuntu Advantage subscription from Canonical.</p>
-        <a href="/pro#livepatch" class="p-button--positive">Get Ubuntu Advantage</a>
+        <p>Get Livepatch with Ubuntu Pro subscription from Canonical.</p>
```

Comment:
I made this change in the copy doc too:

```suggestion
        <p>Get Livepatch with an Ubuntu Pro subscription from Canonical.</p>
```